src/web/docusaurus/docs/getting-started/environment-setup.md

```diff
@@ -288,7 +294,14 @@ Run the `pnpm db:init` script as it will populate the database with an initial s
 `db:seed` is responsible to populate the database with an initial set of data.
 :::
 
-Stop the `parser` service running. If you have Docker app installed locally make sure to stop the `parser` container before continuing to the next step.
+Stop the `parser` service running. Make sure to stop the `parser` container before continuing to the next step by either using
+
+Docker Desktop app:
+![image](https://user-images.githubusercontent.com/58532267/202260945-30cc22f1-a634-4d4c-9d21-55824fd6bb5a.png)
```

Comment:
Let's add this image to `static` folder like https://github.com/Seneca-CDOT/telescope/blob/30731846b8c85ec6f8b5af64f635a8e738e60402/src/web/docusaurus/docs/architecture.md?plain=1#L13